ciao-cli/image.go

```diff
@@ -97,18 +87,11 @@ var imageCommand = &command{
 }
 
 type imageAddCommand struct {
-	Flag            flag.FlagSet
-	name            string
-	id              string
-	containerFormat imageContainerFormat
-	diskFormat      imageDiskFormat
-	minDiskSize     int
```

Comment:
@obedmr I certainly can see an argument for including this as you mentioned.  However, in ciao, launcher will bump the memory and disk size automatically if the workload doesn't specify a large enough memory/disk for the image. So right now controller doesn't use the image mem/disk size at all, but maybe @markdryan might wish it did.